thatkitebot/cogs/uwucog.py

```diff
@@ -12,7 +12,7 @@ async def uwuify(message: str):
     return message.replace('na', 'nya').translate(trans_table).replace("no", "yo").replace("mo", "yo").lower()+' uwu'
 
 # Yes this definately needs its own cog shut the fuck up kite (Jk I love you)
-class UwuCog(commands.Cog, name=""):
```

Comment:
Name was set to "" which caused help command to fail when it tried to send the empty name in a discord embed